test/library/packages/Python/correctness/myTuple.chpl

```diff
@@ -0,0 +1,87 @@
+use Python;
+
+var pyCode = """
+def getTup():
+  return (2, 3, 5, 7)
+
+def getTup2():
+  return (11,)
+""";
+
+proc main() {
+
+  var interp = new Interpreter();
+
+  var pyCodeModule = interp.importModule('__empty__', pyCode);
+  var getTupFunc = pyCodeModule.get('getTup');
+
+  var t = getTupFunc(owned PyTuple);
+  writeln("size ", t.size);
+  writeln("t[0] ", t.get(int, 0));
+  writeln("t[1] ", t.get(1));
+  writeln("t[2] ", t.get(int, 2));
+  writeln("t[3] ", t.get(int, 3));
+
+  // check contains
+  writeln("5 in t: ", t.contains(5));
+  writeln("8 in t: ", t.contains(8));
+
+  // get a slice of the tuple
+  writeln("t[0:2] ", t.get(0..1));
+  writeln("t[1:2] ", t.get(1..1));
+  writeln("t[2:5] ", t.get(2..5));
+  writeln("t[-3:2] ", t.get(-3..1));
+  writeln("t[-3:5] ", t.get(-3..5));
+
+  // get a slice with an empty range
+  writeln("t[1:1] ", t.get(1..0));
+
+  // slice missing at least one bounds
+  writeln("t[2:] ", t.get(2..));
+  writeln("t[:3] ", t.get(..2));
+  writeln("t[:] ", t.get(..));
+
+  // get slice with range and type argument
+  writeln("checking explicit type for slice overload");
+  writeln("t[0:2] ", t.get(owned PyTuple, 0..1));
+  writeln("t[2:5] ", t.get(2*int, 2..5));
+  // Note: calling non-tuple methods on the result of this does not work and it
+  // does not print as a list would.  But you can still make the call
+  writeln("t[0:2] ", t.get(owned PyList, 0..1));
+
```

Comment:
I'm surprised, the PyList implementation uses PySequence calls?

Reply:
The trouble is that the underlying C type returned is not actually a PyList, it's still a PyTuple, we're just treating it as a PyList.  `fromPythonInner` doesn't move the contents into a new PyObject of the specified type, right?

Reply:
Right, but I would still expect something like `PySequence_GetItem` to work, which is what our implementation of PyList does. Anyways, not important to this PR, I was just surprised

Reply:
I think I'm seeing the confusion here.  I meant that I tried `PyList.set` on it.  I suspect `PyList.get` is okay but that's not what I tried

Reply:
Note that I have opened https://github.com/chapel-lang/chapel/issues/26957, which will turn this code into an error if/when we implement that

Reply:
That definitely seems like the right call to me.  Should I remove the line from this test now, then?

Reply:
I think its fine to leave for now